docker/test/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:carbon
+
+LABEL MAINTAINER = "Natalie Elizabeth <natalie.elizabeth@andela.com"
+
+LABEL application="andela-socials"
+
+ENV TERM=xterm-256color
+
+ENV CI=true
+
+RUN apt-get -y update && apt-get -y install xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2
+
+ADD scripts/setup.sh /usr/local/bin/setup.sh
+
+ADD scripts/start.sh /usr/local/bin/start.sh
+
+RUN chmod +x /usr/local/bin/setup.sh
```

Comment:
This is not necessary if we already have `chmod +x` set on the file locally.  That will reduce the image size at best.